chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/urluptime/DatadogSyntheticsTestService.java

```diff
@@ -110,7 +110,7 @@ public SyntheticsAPITest createSyntheticsTest(String url, List<Long> developerId
                                         .type(SyntheticsAssertionType.STATUS_CODE)),
                                 new SyntheticsAssertion(new SyntheticsAssertionTarget()
                                         .operator(SyntheticsAssertionOperator.MATCHES)
-                                        .target("/[\\S\s]+[\\S]+/")
+                                        .target("/[\\S]/")
```

Comment:
I approve the PR, but was wondering if we could make the regex a constant and use it in the job just to keep us honest about using the same values in both places.